src/languages/en.ts

```diff
@@ -4899,6 +4899,22 @@ const translations = {
         date: 'Date',
         time: 'Time',
         none: 'None',
+        settingsStatus: {
```

Comment:
```suggestion
        indicatorStatus: {
```

Probably should be `indicatorStatus`